client/src/components/EmailFrontend.js

```diff
@@ -220,31 +220,34 @@ class EmailFrontend extends React.Component {
                   <span style={{ color: textRed }}>*</span>
                 )}
               </legend>
-              {_.map(field_info.enum_values, (label_by_lang, key) => (
-                <EnumField
-                  classes={
-                    field_info.style && field_info.style === "horizontal"
-                      ? `col-sm-${_.floor(
-                          12 / _.size(field_info.enum_values)
-                        )} inline`
-                      : ""
-                  }
-                  key={`${key}_${field_info.form_type}`}
-                  form_type={field_info.form_type}
-                  label={label_by_lang[lang]}
-                  enum_key={key}
-                  field_id={get_field_id(field_key)}
-                  selected_enums_for_field={completed_template[field_key]}
-                  disabled={disable_forms}
-                  aria-hidden={disable_forms}
-                  state_update_callback={(selected_enums) =>
-                    this.mergeIntoCompletedTemplateState(
-                      field_key,
-                      selected_enums
-                    )
-                  }
-                />
-              ))}
+              <div
+                className={`${
+                  field_info.style &&
+                  field_info.style === "horizontal" &&
+                  "d-flex justify-content-between"
+                }`}
```

Comment:
Replace template literal with `classnames` util, as is this is going to be setting a `className={"false"}` pretty often. OR add a `|| ""` to the end of the expression, but the `classnames` util is just a little cleaner/used more consistently than that.